src/Razor/test/Microsoft.VisualStudio.LanguageServer.ContainedLanguage.Test.Common/Microsoft.VisualStudio.LanguageServer.ContainedLanguage.Test.Common.csproj

```diff
@@ -11,6 +11,7 @@
 
   <ItemGroup>
     <ProjectReference Include="..\..\src\Microsoft.VisualStudio.LanguageServer.ContainedLanguage\Microsoft.VisualStudio.LanguageServer.ContainedLanguage.csproj" />
+    <ProjectReference Include="..\Microsoft.AspNetCore.Razor.LanguageServer.Test.Common\Microsoft.AspNetCore.Razor.LanguageServer.Test.Common.csproj" />
```

Comment:
Needed to expose the `CSharpTestLspServer` class to `TestDocumentManager`, which is in turn needed so we can update the C# backing document's text for provisional completion